contrib/buildrpm.sh

```diff
@@ -46,6 +49,10 @@ if [ $opt_no_dist -eq 1 ]; then
     rpmmacros="$rpmmacros '--undefine=dist'"
 fi
 
+if [ $opt_no_deps -eq 1 ]; then
+    rpmopts="$rpmopts '--nodeps'"
```

Comment:
Why do you need `'` around the option?

Reply:
just for consistency with the rest of the script, where it adds `'` around every parameter to rpmbuild

Reply:
I see `'` around every parameter's option (mostly because of spaces), not the opt name itself.
`rpmopts` before your change:
```
rpmopts="--nodeps --buildroot='${WS}/_rpm'"
```
After the change:
```
rpmopts="--buildroot='${WS}/_rpm' '--nodeps'"
```

Reply:
@amaslenn fixed